packages/experimental/Checkbox/src/Checkbox.types.ts

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { ColorValue } from 'react-native';
+import { FontTokens, IBorderTokens, IForegroundColorTokens, IBackgroundColorTokens } from '@fluentui-react-native/tokens';
+import { IFocusable, IPressableState } from '@fluentui-react-native/interactive-hooks';
+import type { ITextProps, IViewProps } from '@fluentui-react-native/adapters';
+
+export const checkboxName = 'Checkbox';
+
+export interface CheckboxTokens extends FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens {
+  checkboxBackgroundColor?: ColorValue;
+  checkboxBorderColor?: ColorValue;
+  checkmarkColor?: ColorValue;
+  checkmarkVisibility?: number;
+  textBorderColor?: ColorValue;
+  checkboxBorderRadius?: number;
+  checkboxMarginStart?: number;
+  checkboxMarginEnd?: number;
+  checkboxVisibility?: number;
+
+  /**
+   * States that can be applied to a checkbox
+   */
+  disabled?: CheckboxTokens;
+  boxAtEnd?: CheckboxTokens;
+  hovered?: CheckboxTokens;
+  focused?: CheckboxTokens;
+  pressed?: CheckboxTokens;
+  checked?: CheckboxTokens;
+}
+
+export interface CheckboxProps extends Omit<IViewProps, 'onPress'> {
```

Comment:
why omit onPress?

Reply:
I copied this from the old checkbox - I think it's removed because we want them to use 'onChange' instead which is assigned to onPress when we use useAsPressable (see useCheckbox.ts, line 36).